src/OAuthClient/BibClient.php

```diff
@@ -9,13 +9,27 @@
 class BibClient extends APIClient
 {
     /**
-     * @param string $bibId
+     * @param array $bibIds
      * @param $nyplSource
-     * @return null|Bib
+     * @return array|null
      */
-    public static function getBibByIdAndSource($bibId = '', $nyplSource)
+    public static function getBibsByIds(array $bibIds)
     {
-        $url = Config::get('API_BIB_URL') . '/' . $nyplSource . '/' . $bibId;
+        if (count($bibIds) < 1) {
+            APILogger::addError(
+                'Failed',
+                array('No bibIds provided to ' . __FUNCTION__)
+            );
+            return null;
+        } elseif (count($bibIds) > 1) {
+            $bibIdList = implode(",", $bibIds);
+        } else {
```

Comment:
I'm not sure if this is necessary as I'm pretty sure you can call `implode` with an array of one element and then it will just return that one element.